charts/ccsm-helm/values.yaml

```diff
@@ -43,7 +22,47 @@ global:
 #   * [if true] for toggles
 #   * [configuration] for section/group of variables
 
-# Zeebe configuration for the zeebe sub chart. Contains configuration for the broker and gateway pods and related resources.
+# Global configuration for variables which can be accessed by all sub charts
+global:
+
+  # Annotations can be used to define common annotations, which should be applied to all deployments
+  annotations: {}
+  # Labels can be used to define common labels, which should be applied to all deployments
+  labels:
+    app: camunda-cloud-self-managed
+
+  # Image configuration to be used in each sub chart
+  image:
+    # global.image.repository is net set here, otherwise it will overwrite the values from the sub chart
+    #    repository: camunda/zeebe
+
+    # Image.tag defines the tag / version which should be used in the chart
+    tag: 1.3.1
+    # Image.pullPolicy defines the image pull policy which should be used https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy
+    pullPolicy: IfNotPresent
+    # Image.pullSecrets can be used to configure image pull secrets https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
+    pullSecrets: [ ]
+
+  # Elasticsearch configuration which is shared between the sub charts  
+  elasticsearch:
+    # Elasticsearch.disableExporter if true, disables the elastic exporter in zeebe
+    disableExporter: false
+    # Elasticsearch.url can be used to configure the URL to access elasticsearch, if not set services fallback to host and port configuration
+    url:
+    # Elasticsearch.host defines the elasticsearch host, ideally the service name inside the namespace
+    host: "elasticsearch-master"
+    # Elasticsearch.port defines the elasticsearch port, under which elasticsearch can be accessed
+    port: 9200
+    # Elasticsearch.clusterName defines the cluster name which is used by Elasticsearch
+    clusterName: "elasticsearch"
+    # Elasticsearch.prefix defines the prefix which is used by the Zeebe Elasticsearch Exporter to create Elasticsearch indexes
+    prefix: zeebe-record
+  # ZeebeClusterName defines the cluster name for the Zeebe cluster. All pods get this prefix in their name.
+  zeebeClusterName: "{{ .Release.Name }}-zeebe"
```

Comment:
:question: I guess this was already like this before, but does it make sense to have _all_ pods with `-zeebe`? Wouldn't the more common case be to use the release name, and only the Zeebe specific stuff have `-zeebe` in the name? Possibly other sub charts would get `-tasklist` or `-operate` in their name? :man_shrugging: Especially since this is now a CCSM chart and not just a Zeebe chart.

Reply:
You're right. Sorry the comment is a bit wrong actually the pods are named like this:

```
[zell ~/ cluster: zeebe-cluster ns:zell-helm-test]$ kgpo
NAME                                  READY   STATUS              RESTARTS   AGE
elasticsearch-master-0                0/1     Init:0/1            0          15s
elasticsearch-master-1                0/1     Init:0/1            0          15s
elasticsearch-master-2                0/1     Init:0/1            0          15s
test-operate-758bb558bb-bjt2x         0/1     ContainerCreating   0          15s
test-tasklist-7d4b64866c-spd2n        0/1     ContainerCreating   0          15s
test-zeebe-0                          0/1     ContainerCreating   0          15s
test-zeebe-1                          0/1     ContainerCreating   0          15s
test-zeebe-2                          0/1     ContainerCreating   0          15s
test-zeebe-gateway-785997b975-9nstg   0/1     Running             0          15s
```